connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -427,7 +427,7 @@ export async function syncZendeskTicketBatchActivity({
     return { hasMore: false, nextLink: "" };
   }
 
-  const users = await zendeskApiClient.users.list();
+  const users = (await zendeskApiClient.users.list()) || [];
```

Comment:
can `.list()` return undefined ?

Reply:
not in the types, yes in the actual implementation